.coderabbit.yaml

```
@@ -12,12 +12,23 @@ reviews:
     drafts: false
     # Empty base_branches array means review ALL PRs regardless of target branch
     base_branches: [develop, release]
-    labels: []
+    # Only review PRs with the "Needs Review" label
+    labels: ["Needs Review"]
     ignore_title_keywords: []
 
   # Generate high-level summaries for pull requests
   high_level_summary: true
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)